plugins/maciVoting/services/coordinator/utils/types.ts

```diff
@@ -0,0 +1,349 @@
+import { type EPolicies, type EInitialVoiceCreditProxies, type EMode } from "@maci-protocol/sdk";
+
+import type { Hex } from "viem";
+import { type ESupportedNetworks } from "./networks";
+
+/**
+ * IConstantInitialVoiceCreditProxyArgs represents the arguments for deploying a constant initial voice credit proxy
+ */
+export interface IConstantInitialVoiceCreditProxyArgs {
+  /**
+   * The amount of initial voice credits to deploy
+   */
+  amount: number;
+}
+
+/**
+ * IEASPolicyArgs represents the arguments for deploying an EAS policy
+ */
+export interface IEASPolicyArgs {
```

Comment:
We do not need the `IEASPolicyArgs`

Reply:
Removed all policies as deleted the types file